maps/randomvaults/spessmart.dm

```diff
@@ -64,16 +64,16 @@ var/list/shop_prices = list( //Cost in space credits
 /obj/item/weapon/bonegel = 30,
 /obj/item/weapon/FixOVein = 30,
 
-/obj/item/weapon/switchtool/surgery = 250,
-/obj/item/weapon/switchtool/swiss_army_knife = 500,
+/obj/item/weapon/switchtool/surgery = 100,
+/obj/item/weapon/switchtool/swiss_army_knife = 300,
 /obj/item/weapon/rcl = 50,
-/obj/item/weapon/glue = 500,
+/obj/item/weapon/glue = 400,
 /obj/item/weapon/chisel = 20,
 /obj/item/weapon/scythe = 50,
-/obj/item/bluespace_crystal/flawless = 10000,
-/obj/item/bluespace_crystal/artificial = 500,
-/obj/item/bluespace_crystal = 1000,
-/obj/item/device/assembly_frame = 50,
+/obj/item/bluespace_crystal/flawless = 500,
```

Comment:
that's a hug price drop

Reply:
The flawless crystal is probably the only really egregious thing here, because it allows infinite use of the tunneler which is nutty.